From 34b9fad5266d91af7706d73f3fc707e92be8069d Mon Sep 17 00:00:00 2001
Message-Id: <34b9fad5266d91af7706d73f3fc707e92be8069d.1267106475.git.paul.gortmaker@windriver.com>
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Wed, 24 Feb 2010 21:06:42 -0500
Subject: [PATCH 1/5] MILS shim: clarify demux of VBI vs VDK

Do the demux up at the top of the file, where you can see
it when looking for it.

This will eventually be folded into the shim commit to
reduce the overall commit quantity/noise.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 include/vbi/interface.h |    9 +++++----
 1 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/include/vbi/interface.h b/include/vbi/interface.h
index 0e62485..a78e221 100644
--- a/include/vbi/interface.h
+++ b/include/vbi/interface.h
@@ -80,7 +80,10 @@ identity mapped.
 
 */
 
-#if !defined(_VBI_INTERFACE_H) && !defined(CONFIG_WRHV_MILS)
+#ifdef CONFIG_WRHV_MILS
+#include <vbi/vdk_interface.h>
+#else
+#ifndef _VBI_INTERFACE_H
 #define _VBI_INTERFACE_H
 
 #include <vbi/types.h>
@@ -582,7 +585,5 @@ struct vb_config
 };
 
 #endif /*_ASMLANGUAGE */
-
-#else
-#include <vbi/vdk_interface.h>
 #endif  /* _VBI_INTERFACE_H */
+#endif /* ! CONFIG_WRHV_MILS */
-- 
1.6.5.2

